frameworks/C/nginx/njs/hello.js

```diff
@@ -0,0 +1,11 @@
+function hello(r) {
```

Comment:
The file is not formatted consistently

Reply:
I copied from njs-examples:
https://github.com/nginx/njs-examples#hello-world-example-httphello